kani-driver/src/harness_runner.rs

```diff
@@ -64,6 +64,12 @@ impl<'sess> HarnessRunner<'sess> {
                         &harness,
                     )?;
 
+                    if self.sess.args.synthesize_loop_contracts {
+                        println!("Start to synthesize loop contracts for all loops.");
```

Comment:
Please check if we are on quiet mode before printing this message.

Reply:
Add